patronload/database.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import oracledb
+
+
+def build_sql_query(fields: list, table: str) -> str:
```

Comment:
Again, I would double-check all the type hints to see where we can be more specific. This should really be `list[str]` because we require that the list only contain string values.